src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,52 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
-    private final StorageDao storageDao = new StorageDaoImpl();
+    public static final int MAX_AGE = 100;
+    public static final int MIN_AGE = 18;
+    public static final int MIN_LENGTH = 6;
+    private static final StorageDao storageDao = new StorageDaoImpl();
```

Comment:
```suggestion
    private final StorageDao storageDao = new StorageDaoImpl();
```
let's not make storageDao static